aksel.nav.no/website/components/website-modules/toc/TableOfContents.tsx

```diff
@@ -27,7 +27,7 @@ function TableOfContents({ toc, variant }: TableOfContentsProps) {
   } as React.CSSProperties;
 
   return (
-    <aside
+    <div
```

Comment:
https://dequeuniversity.com/rules/axe/4.10/landmark-complementary-is-top-level
> Aside should not be contained in another landmark

Should we use `<nav>`?

Reply:
> We could perhaps disable the rule for now, and handle it in a separate issue. 

Like this approach 👍